crates/noirc_driver/src/lib.rs

```diff
@@ -43,11 +43,15 @@ pub struct CompileOptions {
     /// Display output of `println` statements
     #[arg(long)]
     pub show_output: bool,
+
+    /// Compile and optimize using the new experimental SSA pass
+    #[arg(long)]
```

Comment:
Lets make the default value false here

Reply:
Hmm actually we are doing defaults in a different way than usual. This is then unrelated to the PR